README.md

```diff
@@ -1 +1,249 @@
-# vit-clinic
\ No newline at end of file
+# vit-clinic
+
+## Getting Started
+
+This repository includes files with plain SQL that can be used to recreate a database:
+
+- Use [schema.sql](./schema.sql) to create all tables.
+- Use [data.sql](./data.sql) to populate tables with sample data.
+- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
+
+<a name="readme-top"></a>
+
```

Comment:
- [ ] [OPTIONAL]
Let's endeavor to remove or update the underlined aspects of the ReadMe to make the ReadMe appear more professional!

![image](https://github.com/mhnazary/vit-clinic/assets/12745474/2495e358-dd12-4865-81aa-05d5a9dbd7b6)